utils/appliance/__init__.py

```diff
@@ -1645,6 +1643,32 @@ def reset_automate_model(self):
             ssh_client.run_rake_command("evm:automate:reset")
 
 
+@navigate.register(IPAppliance)
+class LoggedIn(NavigateStep):
+    def step(self):
+        from cfme.login import login_admin
+        from utils.browser import browser
+        browser()
+        login_admin()
```

Comment:
Well, we used to handle it with the context manager I think, so you'd just navigate inside the context manager......but yes we could also offer up this solution, or you set the user on the appliance object as the user it should use to log in before you navigate.